pkg/trace/api/profiles.go

```diff
@@ -68,7 +71,17 @@ func (r *HTTPReceiver) profileProxyHandler() http.Handler {
 	if err != nil {
 		return errorHandler(err)
 	}
-	tags := fmt.Sprintf("host:%s,default_env:%s", r.conf.Hostname, r.conf.DefaultEnv)
+	tags := fmt.Sprintf("host:%s,default_env:%s,agent_version:%s", r.conf.Hostname, r.conf.DefaultEnv, info.Version)
+	if r.conf.IsFargate {
```

Comment:
This branch is not tested. It should be easy to simply set this config option to true. The return value of `GetOrchestrator` can remain `Unknown` for the sake of testing. Or, if you wish for it to have the `EKS` value you can just set the `eks_fargate` config option.

Reply:
added tests for both the fargate=false path and the fargate=true path and test for orchestrator tag